dist/molkobain-datacenter-view/molkobain-datacenter-view/sv.dict.molkobain-datacenter-view.php

```diff
@@ -0,0 +1,99 @@
+	'Class:Location/Attribute:parent_id' => 'Huvudplats',
+	'Class:Location/Attribute:parent_id+' => 'plats som hostar denna (ex. för ett \'rum\', skal vara en \'våning\' eller en \'byggnad\')',
+	'Class:Location/Attribute:locations_list' => 'Inkluderade platser',
+	'Class:Location/Attribute:locations_list+' => 'Lista över alla platser som är inkluderade i denna',
+	// - PhysicalDevice (default entries for custom classes)
+	'Class:PhysicalDevice/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PhysicalDevice/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'element'),
+	'Class:PhysicalDevice/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	// Note: Physical device can be either an host (eg. enclosure) or a device (eg. server), so we use a generic sentence
+	'Class:PhysicalDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:host'], 'element', 'dess host'),
+	// - Rack
+	'Class:Rack/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:Rack/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'rack'),
+	// - Enclosure
+	'Class:Enclosure/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:Enclosure/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'chassi'),
+	'Class:Enclosure/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:Enclosure/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:host'], 'chassi', 'rack'),
+	'Enclosure:baseinfo' => sprintf($aMolkobainThesaurus['Fieldset:baseinfo']),
+	'Enclosure:moreinfo' => sprintf($aMolkobainThesaurus['Fieldset:moreinfo']),
+	'Enclosure:otherinfo' => sprintf($aMolkobainThesaurus['Fieldset:otherinfo']),
+	'Enclosure:dates' => sprintf($aMolkobainThesaurus['Fieldset:dates']),
+	// - Datacenter device
+	'Class:DatacenterDevice/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:DatacenterDevice/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'enhet'),
+	'Class:DatacenterDevice/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:DatacenterDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'enhet'),
+	// - PDU
+	'Class:PDU/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PDU/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'enhet'),
+	'Class:PDU/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:PDU/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'Strömenhet'),
+));
+
+// UI
+Dict::Add('EN US', 'English', 'English', array(
```

Comment:
```suggestion
Dict::Add('SV SV', 'Swedish', 'Svenska', array(
```

Typo, the old "english" header was kept.